src/index.js

```diff
@@ -119,7 +124,7 @@ export default class Fetchja {
       })
 
       // Fetch
-      return fetch(url, {
+      return fetchFunction(url, {
```

Comment:
Why not `this.fetchFunction`? Don't know if we need `const fetchFunction`.

Reply:
I trust your judgement. Thank you!